scrapyelasticsearch/scrapyelasticsearch.py

```diff
@@ -101,13 +101,14 @@ def process_unique_key(self, unique_key):
         return unique_key
 
     def get_id(self, item):
-        item_unique_key = item[self.settings['ELASTICSEARCH_UNIQ_KEY']]
-        unique_key = self.process_unique_key(item_unique_key)
```

Comment:
what about passing in the key as a list? so the change looks like:

```
item_unique_key = item[self.settings['ELASTICSEARCH_UNIQ_KEY']]
if isinstance(item_unique_key, list):
    item_unique_key = '-'.join(item_unique_key)
```

Reply:
I think it is ok, if you prefer that way, you can change it that way. Do you plan to include the feature and update on pip repository ?

Reply:
Yeah, I personally think it is cleaner that way. I will make the change and cut a release tonight.